### DISCUSSION TITLE
neo4j :sysinfo: 报错

### DISCUSSION THREAD

**Discussion by Drun1baby** [tabby-sec/tabby#48] [Q&A] [answered]
tabby 是正常跑起来了， `CALL tabby.help('all')` 命令没问题

`:sysinfo` 报错如图

![image](https://user-images.githubusercontent.com/75228348/221747804-dda7b7c5-7ba6-4299-bc2f-4dabed23d7d1.png)

**Comment by wh1t3p1g** [comment, marked as answer]
我也有这个错误，但是不影响使用

**Reply by Drun1baby** [reply]
> 我也有这个错误，但是不影响使用

确实如此，已解决

谢谢师傅！
